packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/selects.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { SelectField } from '.';
+import { TSelectsFieldNames, TDropdownItems } from '../quick-strategy.types';
+import { TSelects } from './components.types';
+
+const Selects = ({
+    field_name,
+    id,
+    is_mobile,
+    dropdown_list,
+    selected_value,
+    label,
+    select_value,
+    setFieldValue,
+    className,
+    is_able_disabled,
+    values,
+    onChangeDropdownItem,
+    onHideDropdownList,
+    onScrollStopDropdownList,
+    selected_trade_type,
+    selected_symbol,
+    is_input_field,
+}: TSelects & { is_input_field: boolean }) =>
+    !is_input_field && !(id === 'duration-unit') ? (
```

Comment:
``(!is_input_field && !(id === 'duration-unit'))` && ` can we use this like this and remove the ternary ?

Reply:
@rupato-deriv no, because JSXElement(component) requires something returned anyway, we don’t want to see any warnings